OpenRA.Mods.Common/Traits/Power/CanPowerDown.cs

```diff
@@ -20,9 +20,13 @@ public class CanPowerDownInfo : UpgradableTraitInfo, Requires<PowerInfo>
 		public readonly bool CancelWhenDisabled = false;
 
 		public readonly string IndicatorImage = "poweroff";
-		public readonly string IndicatorSequence = "offline";
+		[SequenceReference("IndicatorImage")] public readonly string IndicatorSequence = "offline";
 
-		public readonly string IndicatorPalette = "chrome";
+		[PaletteReference] public readonly string IndicatorPalette = "chrome";
+
+		public readonly string NotificationType = "Sounds";
```

Comment:
This isn't a problem in literally every other trait (except PlaceBeacon) that uses speech and/or notifications.

In the current pr it is not possible to play a powerdown/powerup sound (like machinery turning on/off) at the structure you are toggling at the same time as playing an EVA notification.  The changes that I have requested makes this situation doable – an improvement – as well as bringing the code in line with the rest of our traits.